db/seeds.rb

```diff
@@ -29,3 +29,33 @@
   user.password_confirmation = "password"
   user.name = "Test User 3"
 end
+
+# Create hardcoded test topics
+Topic.find_or_create_by(name: "Test Topic 1")
+
+Topic.find_or_create_by(name: "Test Topic 2")
+
+Topic.find_or_create_by(name: "Test Topic 3")
+
+# Create hardcoded test targets
+
+Target.find_or_create_by(title: "Test Target 1") do |target|
+  target.user_id = 1
```

Comment:
Instead of hardcoding the id, you can assign the user1 to a variable and then use it here, the same for topics